packages/starlight/package.json

```diff
@@ -168,7 +168,9 @@
       "default": "./integrations/expressive-code/hast.mjs"
     },
     "./index.astro": "./index.astro",
+    "./indexSSR.astro": "./indexSSR.astro",
     "./404.astro": "./404.astro",
+    "./404SSR.astro": "./404SSR.astro",
```

Comment:
@HiDeoo mentioned a question around naming of the different entrypoints. I don’t have a strong opinion, but one idea would also be to move all four into a dedicated `routes/` directory to help us keep organised. Up to you @Fryuni if you want to do that or not — it’s not essential and can also happen later in some work tidying up the repo.